packages/trpc/server/routers/viewer/eventTypes.tsx

```diff
@@ -139,7 +139,7 @@ export const eventTypesRouter = createProtectedRouter()
           },
         });
 
-        if (!hasMembership) {
+        if (!hasMembership || hasMembership.role === "MEMBER") {
```

Comment:
```suggestion
        if (!["ADMIN", "OWNER"].includes(hasMembership?.role)) {
```

Let's be explicit on WHO has these permissions